Library/Formula/cctools.rb

```diff
@@ -11,6 +11,7 @@ class Cctools < Formula
   end
 
   bottle do
+    does_not_need_relocation
     cellar :any
```

Comment:
It would be nice if we can combine these two annotations as one like `cellar :skip_relocation` or something.